src/Appwrite/Migration/Version/V19.php

```diff
@@ -651,7 +651,7 @@ protected function fixDocument(Document $document): Document
         switch ($document->getCollection()) {
             case 'attributes':
             case 'indexes':
-                $status = $document->getAttribute('status', '');
+                $status = $document->getAttribute('status', $document->getAttribute('status', ''));
                 if ($status === 'failed') {
                     $document->setAttribute('error', 'Unknown problem');
                 }
```

Comment:
This isn't required, but checking the default for the error attribute is:

```suggestion
                $status = $document->getAttribute('status');
                if ($status === 'failed') {
                    $document->setAttribute('error', $document->getAttribute('error', 'Unknown problem'));
                }
```